zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -2592,6 +2595,7 @@ search.toolbar.label.type.header = Header
 search.toolbar.label.type.request = Request
 search.toolbar.label.type.response = Response
 search.toolbar.label.type.url = URL
+search.toolbar.placeholder.regex = Type regular expression
```

Comment:
We should also agree how the prompts should look like, e.g. why "type" here and not in the other field? IMO that you need to type a value it's already implied.

Reply:
My comment is about consistency generally and to the inclusion of "Type" particularly. Compare with the other prompt added above which just says "Search expression" (why not here e.g. "Search with regular expression"?).

Reply:
> 'Search term or regular expression' 

?